frappe/public/js/frappe/form/sidebar/assign_to.js

```diff
@@ -130,81 +124,126 @@ frappe.ui.form.AssignTo = Class.extend({
 
 frappe.ui.form.AssignToDialog = Class.extend({
 	init: function(opts){
-		var me = this
-		var dialog = new frappe.ui.Dialog({
-			title: __('Add to To Do'),
-			fields: [
-				{ fieldtype: 'Link', fieldname: 'assign_to', options: 'User', label: __("Assign To"), reqd: true, filters: { 'user_type': 'System User' }},
-				{ fieldtype: 'Check', fieldname: 'myself', label: __("Assign to me"), "default": 0 },
-				{ fieldtype: 'Small Text', fieldname: 'description', label: __("Comment") },
-				{ fieldtype: 'Section Break' },
-				{ fieldtype: 'Column Break' },
-				{ fieldtype: 'Date', fieldname: 'date', label: __("Complete By") },
-				{ fieldtype: 'Column Break' },
-				{ fieldtype: 'Select', fieldname: 'priority', label: __("Priority"),
-					options: [
-						{ value: 'Low', label: __('Low') },
-						{ value: 'Medium', label: __('Medium') },
-						{ value: 'High', label: __('High') }
-					],
-					// Pick up priority from the source document, if it exists and is available in ToDo
-					'default': ["Low", "Medium", "High"].includes(opts.obj.frm && opts.obj.frm.doc.priority
-						? opts.obj.frm.doc.priority : 'Medium')
-				},
-			],
-			primary_action: function() { frappe.ui.add_assignment(opts, this) },
-			primary_action_label: __("Add")
-		})
-		$.extend(me, dialog);
-
-		me.dialog = dialog;
-
-		me.dialog.fields_dict.assign_to.get_query = "frappe.core.doctype.user.user.user_query";
-
-		var myself = me.dialog.get_input("myself").on("click", function() {
-			me.toggle_myself(this);
+		$.extend(this, opts);
+
+		this.make();
+		this.set_description_from_doc();
+	},
+	make: function() {
+		let me = this;
+
+		me.dialog = new frappe.ui.Dialog({
+			title: __('Add to ToDo'),
+			fields: me.get_fields(),
+			primary_action_label: __("Add"),
+			primary_action: function() {
+				let args = me.dialog.get_values();
```

Comment:
```suggestion
							let txt = this.$input.val();
							data = this.df.get_data(txt);
```
https://github.com/frappe/frappe/pull/10412

Reply:
@hrwX  not needed. get_values do there work